src/pages/blog/03-honey-flow/index.md

```diff
@@ -0,0 +1,39 @@
+---
+title: "Start Honey flowing again"
+date: "2020-10-13"
+author: "Seeds"
+featuredImage: ./land.png
+previewText: "Let the farming games begin!"
+---
+
+Two weeks have passed and the community has had some time to consolidate. There are a lot of new faces and new contributors from different regions. After having to stop Conviction Voting due to a misconfiguration these last two weeks were a blessing in disguise that gave the necessary breathing space after the explosion of growth we had.
+
+The process to submit new proposals was updated and a new social expectation started to grow towards better moderation. The documentation grew significantly with community members jumping to create onboarding guides, FAQ, video tutorials and translations. The Discord server was reorganized with different roles for each swarm to improve our daily communications.
+
+The pollen distribution using SourceCred is a success, new community members are excited about being rewarded for their contributions. We have a [forum post](https://forum.1hive.org/t/new-pollen-proposal-for-period-starting-oct-23/407/) with a proposal to keep funding the initiative in future weeks that saw a lot of engagement and discussion.
+
+## When farming?
+
+![Cred to Detzy for this amazing meme](./when.jpg)
```

Comment:
I think in this format we cannot link the Actual meme post on Discord.
We could alternatively do html

```
<div style="text-align:center; margin-bottom: 24px;">
  <img src="./when.jpg" />
  <span style="font-style: italic; font-size: 15px;">
    Cred to Detzy for 
    <a href="https://discord.com/channels/698287700834517064/743890898781339658/764545821651763232">
      this amazing meme
    </a>
  </span>
</div>
```